data/json/items/gunmod/underbarrel.json

```diff
@@ -237,6 +237,21 @@
     }
   },
   {
+    "id": "sks_bayonet",
+    "copy-from": "spike",
+    "type": "GENERIC",
+    "name": "sks bayonet",
```

Comment:
Why not call it something generic like 'integrated bayonet' so that we can use it on other guns with built-in pokers as well?